examples/plot_skydriver_scan_result.py

```diff
@@ -36,11 +37,14 @@ def main() -> None:
     args = parser.parse_args()
 
     rc = get_rest_client()
-    serialzed = rc.request_seq("GET", f"/scan/{args.scan_id}/result")["skyscan_result"]
+    serialized = rc.request_seq("GET", f"/scan/{args.scan_id}/result")["skyscan_result"]
 
-    result = SkyScanResult.deserialize(serialzed)
-    result.create_plot()
-    result.create_plot_zoomed(plot_bounding_box=True)
+    plotter = SkyScanPlotter()
+
+    result = SkyScanResult.deserialize(serialized)
+
+    plotter.create_plot(result)
+    plotter.create_plot_zoomed(result, plot_bounding_box=True)
```

Comment:
I wonder if it would be better to rename this to `create_healpy_cartview`

Reply:
It is a bit jargon-ish and not very descriptive, but I agree we may need a better naming overall.